### DISCUSSION TITLE
Can't Access Dashboard or Student Pack despite application approval

### DISCUSSION THREAD

**Discussion by DoodleMochi** [community/community#55326] [GitHub Education] [answered]
# Hi everyone,

Like the student here, #41865 , I can't access the benefits page. However, I also can't access the "dashboard" ([https://education.github.com/globalcampus/student](url)), as I get redirected to the [https://education.github.com/](url) homepage as seen here.
![image](https://github.com/community/community/assets/106620795/4b106548-e6a9-4d3a-99c3-0b9e23fdac19)
Like the student in the discussion linked above, I have been verified for a full day, without any access to student resources. The only thing that indicates my verification is the re-verification page.
![image](https://github.com/community/community/assets/106620795/454be9cf-6c45-4228-bf4a-c5eddc55a7b5)
I didn't even get an email confirming my verification, despite being able to receive emails about previous application rejections.
What should I do?

**Comment by Ntshangase** [comment]
- Just try logging out of GitHub then Login again.

**Comment by DanVasii** [comment, marked as answer]
@DoodleMochi did you get the access to the pack ? I am facung the same issue. If yes, how long did it take to get access?

**Reply by DoodleMochi** [reply]
It took 4-5 days, which is far longer than a reasonable period. Best of luck for yours.
